pkgs/by-name/us/usque/package.nix

```
@@ -0,0 +1,51 @@
+{
+  fetchFromGitHub,
+  buildGoModule,
+  lib,
```

nit: generally packages start with lib, builder, fetcher
```suggestion
{
  lib,
  buildGoModule,
  fetchFromGitHub,
```